src/WebApi/Program.cs

```diff
@@ -30,7 +32,18 @@
     app.UseHsts();
 }
 
+// Basic Healthy/Degraded/Unhealthy result
```

Comment:
Is it worth wrapping both of these `UseHealthChecks()` into their own extension method that takes care of everything?

Reply:
Done